src/drafts/InlineAutocomplete/InlineAutocomplete.tsx

```diff
@@ -79,7 +79,6 @@ const InlineAutocomplete = ({
   sx,
   children,
   tabInsertsSuggestions = false,
-  // Forward accessibility props so it works with FormControl
```

Comment:
The context-based approach means `InlineAutocomplete` and `Autocomplete` no longer have to do anything special at all to work with `FormControl`, because they both wrap `TextInput`/`Textarea` which call this hook and get the context themselves. 

We still forward the rest props for `InlineAutocomplete` to avoid a breaking change (it could be useful in other situations).